flow/platforms/sky130hd_fakestack/make_tracks.tcl

```diff
@@ -0,0 +1,10 @@
+make_tracks li1 -x_offset 0.23 -x_pitch 0.46 -y_offset 0.17 -y_pitch 0.34
```

Comment:
Do you still need this file or does your make_tracks change suffice?

Reply:
I verified that the make_tracks  works fine.  It produces the same results as with and without the make_tracks.tcl file -- so should we just remove this file?  I think it makes sense to remove the file

Reply:
Please remove it